http/web.go

```diff
@@ -349,11 +363,12 @@ func (w *loggingWriter) Done() {
 		slog.Duration("duration", time.Since(w.Start)),
 		slog.Int("statuscode", w.StatusCode),
 		slog.String("proto", strings.ToLower(w.R.Proto)),
-		slog.Any("remoteaddr", w.R.RemoteAddr),
+		slog.String("remoteaddr", w.remoteAddr()),
```

Comment:
We could log the request.RemoteAddr too, as `original-address` or something, but it's fundamentally not very interesting. It might have a marginal use if you're running a cluster of frontend proxies talking to the backend though, so I don't mind if you want it.